app/build.gradle

```diff
@@ -5,7 +5,7 @@ apply from: './jacoco.gradle'
 android {
 
     compileSdkVersion rootProject.compileSdkVersion
-    buildToolsVersion rootProject.buildToolsVersion
+    buildToolsVersion '28.0.3'
```

Comment:
Why have you removed the variable ?

Reply:
That's automatically done by Android Studio. I will change that.

Reply:
So check the changes using the files changed section and fix them after you make the PR.